include/zephyr/settings/settings.h

```diff
@@ -619,6 +619,48 @@ int settings_runtime_commit(const char *name);
  */
 int settings_storage_get(void **storage);
 
+#ifdef CONFIG_SETTINGS_STATIC_DATA
+/**
+ * @struct settings_data_static
+ * Settings data static structure. This can be used to place compile time
+ * data in a format that can be used by the settings subsystem. The settings
+ * static data is registered in a linker section by using a call to
+ * SETTINGS_STATIC_DATA_DEFINE().
+ */
+struct settings_static_data {
+
+	const char *name;
+	/**< Name of data. */
+
+	const void *data;
+	/**< Pointer to data */
+
+	size_t size;
+	/**< Data size */
```

Comment:
description goes above type

Reply:
This syntax is perfectly valid (https://www.doxygen.nl/manual/docblocks.html#memberdoc), but oddly used here. One would typically use it to comment after the element, but on the same line.